### ISSUE TITLE
Add more than five options to widgets

### ISSUE THREAD

**Issue by kristjanbjarni** [EdgeTX/edgetx#1840]
Creating complex widgets scripts is very limited currently because of only having maximum of five available options for customizing widgets. I would like to see this maximum raised to allow for more options.

**Comment by pfeerick** [comment]
I would have sworn there was an issue for this already, but it seems like there isn't.  In my own experience I already hit the limit of options for internal widgets so couldn't implement https://github.com/EdgeTX/edgetx/issues/1095 yet... so wanted to improve that. I'm thinking the simplest way is to make it a fully-screen window, this allow it to scroll normally and such... as the current popup has issues, and really isn't necessary as it's not like it can update dynamically as you change settings.

**Comment by kristjanbjarni** [comment]
Yes, full screen makes much more sense. This is a really limiting factor for creating useful and easy to use widgets.

**Comment by Eldenroot** [comment]
https://github.com/opentx/opentx/pull/8052 maybe these too?

**Comment by pfeerick** [comment]
Er... 1x2, 1x3, 1x4 layouts already exist in ETX since 2.5... And this is off topic,

**Comment by raphaelcoeffic** [comment]
The main issue regarding widget options is that for now they use a fixed size array in memory, thus increasing the memory consumption for all models exponentially. So we would need to switch to a list with dynamic allocation instead, which is kind of non-trivial right now. But we should, indeed.

**Comment by raphaelcoeffic** [comment]
Displaying these options is not the issue, we shall find a way ;-)

**Comment by philmoz** [comment]
> The main issue regarding widget options is that for now they use a fixed size array in memory, thus increasing the memory consumption for all models exponentially.

Is memory really a problem, as this change only applies to color radios?

I tried increasing the widget option count to 10 on my TX16S.
RAM usage increased to ~78K.

AFAIK all color radios have 192K RAM so this is still using less than 1/2 what is available.

Dynamic memory (malloc/new) comes out of the heap which is in SDRAM.
